StormyHares/BangazonApi
# Orders products join mw

## Ticket Link:
  - #10

## Description:
  - Added the ability to get product details when running a get request and getsingleorder request for orders. 

## Steps to Test:

 1. Make sure you have a fully populated database, especially in regards to all tables related to orders. 

1. Get all orders and confirm that the product details are displayed properly. 

1. Get a single order and confirm that the product details are displayed properly.
